src/main/java/mate/academy/spring/boot/model/Book.java

```diff
@@ -77,13 +77,13 @@ public void setCoverImage(String coverImage) {
     @Override
     public String toString() {
         return "Book{"
-                + "id=" + id
-                + ", title='" + title + '\''
-                + ", author='" + author + '\''
-                + ", isbn='" + isbn + '\''
-                + ", price=" + price
-                + ", description='" + description + '\''
-                + ", coverImage='" + coverImage + '\''
+                + "id= " + id
+                + ", title= '" + title + '\''
+                + ", author= '" + author + '\''
+                + ", isbn= '" + isbn + '\''
+                + ", price= " + price
+                + ", description= '" + description + '\''
+                + ", coverImage= '" + coverImage + '\''
```

Comment:
Add constraints for fields, also add `@Table`
In this stage, did you learn Lombok library? If yes, use annotations instead of get and set methods